platform/wayland/cog-view-wl.c

```diff
@@ -188,8 +187,7 @@ cog_wl_view_create_backend(CogView *view)
 #endif
     };
 
-    self->exportable =
-        wpe_view_backend_exportable_fdo_egl_create(&client, self, viewport->window.width, viewport->window.height);
```

Comment:
From now on the view will get the right geometry  once the View were added to the Viewport (which is the one who has the Window dimensions). This takes place in: https://github.com/Igalia/cog/pull/666/files#diff-bd42155d30f2b7fed99fdf7a4c9f75f78f54a49509ccf0f6feabb22b20af30e7R391